scripts/deploy.sh

```diff
@@ -125,13 +134,21 @@ function force_release_enabled() {
 	return 1
 }
 
+function cleanup() {
+	if is_enabled "${SKIP_CLEANUP}" ; then
+		anounce "SKIP_CLEANUP is set to ${SKIP_CLEANUP}, skipping this step"
```

Comment:
anounce? spelling? don't think I've seen that command before either way

Reply:
That's an existing function in the script

Reply:
`s/anounce/announce/g`

Reply:
Pushed a spelling fix